content/docs/started/getting-started-gke.md

```diff
@@ -345,6 +345,20 @@ cd ${KFAPP}
 ${KUBEFLOW_REPO}/scripts/kfctl.sh apply k8s
 ```
 
+### GCP Service Accounts
+
+Creating a deployment using `kfctl.sh` creates three service accounts in the GCP Project. These service accounts are created using the principle of least privilege. The three service accounts are:
+
+* kubeflow-admin
```

Comment:
Won't the service account names depend on the KFAPP(deployment) name?

Reply:
Updated.